CHANGELOG.md

```diff
@@ -35,6 +35,7 @@
 - ODS AMI build failing due an E2E test error of ionic quickstarter ([#742](https://github.com/opendevstack/ods-quickstarters/issues/742))
 - ODS AMI build failing due an missing list of supported browsers in ionic quickstarter ([#756](https://github.com/opendevstack/ods-quickstarters/issues/756))
 - inf-terraform-aws: Fix error handling of Makefile ([#680](https://github.com/opendevstack/ods-quickstarters/issues/680))
+- Removes jcenter repositories and jdk-17 usage ([#807](https://github.com/opendevstack/ods-quickstarters/pull/807))
```

Comment:
isn't this also a breaking change? If so I'd move this entry to `### Modified`. Also I'd rename to `- Remove JCenter repositories and jdk 17 support ...` or so

Reply:
I've changed it. It was an initial message. Now it is better explained.
Do you see it ok now?

Reply:
Yes. You're right. It was a breaking change.
But we found a different solution that keeps jdk 17, using Adoptium Temurin, maintained by Eclipse.
More information in the following link:
https://github.com/opendevstack/ods-quickstarters/issues/808